app/controllers/course/forum/forums_controller.rb

```diff
@@ -1,7 +1,8 @@
 # frozen_string_literal: true
 class Course::Forum::ForumsController < Course::Forum::Controller
   before_action :load_forum, except: [:index, :new, :create, :search]
-  load_resource :forum, class: Course::Forum.name, through: :course, only: [:index, :new, :create]
+  load_and_authorize_resource :forum, class: Course::Forum.name, through: :course,
```

Comment:
@fonglh Can you check if there's any existing forum been filtered out ?  I am not sure why it's load_resource before.